src/render.js

```diff
@@ -55,7 +55,7 @@ export const renderNode = (
   }
   if (node.entity !== null) {
     const entity = entityMap[node.entity];
-    const entityRenderer = entity && (entityRenderers[entity.type] || entityRenderers.fallbackRenderer)
+    const entityRenderer = entity && (entityRenderers[entity.type] || entityRenderers.fallbackRenderer || (node.block.type === ATOMIC_BLOCK_TYPE && entityRenderers.atomicFallbackRenderer))
```

Comment:
let's remove the entityRenderers.fallbackRenderer. It's breaking and all. But only you and us are using redraft. And we are not using it.